packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -183,120 +176,115 @@ test.describe('Prop: `hasBackButton`', () => {
 });
 
 test.describe('Prop: `heading`', () => {
-    test('should display & render long headings correctly', async ({ page, mount }) => {
-        await mount(PieModal, {
-            props: {
-                heading: 'This is a modal heading but super long and should span multiple lines, hopefully this should never happen on production!',
-                isOpen: true,
-                hasBackButton: true,
-                isDismissible: true,
-                leadingAction: {
-                    text: 'Confirm',
-                },
-            } as ModalProps,
-        });
-
-        const modal = page.locator(componentSelector);
-        await expect.soft(modal).toBeVisible();
+    test('should display & render long headings correctly', async ({ page }) => {
+        const modalDefaultPage = new ModalDefaultPage(page);
+        const props: Partial<ModalProps> = {
+            heading: 'This is a modal heading but super long and should span multiple lines - hopefully this should never happen on production',
```

Comment:
I had to modify this due to a Regex rule in Storybook that disables certain characters when provided as part of the URL query param to set prop values.

https://github.com/storybookjs/storybook/issues/14225